packages/documentation/docs/controls/_pane_styleguide.md

```diff
@@ -0,0 +1,68 @@
+Panes are interactive components that allow users to access hidden content or options that are not visible on the screen. 
+
+It can help users to focus better on their tasks since related controls are visually grouped and the main content contains less information. They are beneficial to use for more compact and hierarchically better organized content, providing a more dynamic layout. You can use them to give contextual information or providing a more tidied up main content for your users for a better task focus.
+
+![Pane overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=1680-22044&mode=design&t=iP7h44Wf17P209P7-4)
+
+1. Left pane (inline & expanded)
+2. Top pane (inline)
+3. Right pane (floating)
+4. Bottom pane (inline)
+
+## Variants
+Up to four floating or inline panes can be placed in a panel layout component.
+
+- **Floating**: The panes are placed above the main content but below the nav menu and header.
+- **Inline**: The panes are placed on one level with the main content. When expanded, they move the main content and reduce its available space.
+
+## Options
+
+- **Heading**: Set a headline for the pane. We typically use a short name that describes its content.
+- **Icon**: An icon can be displayed on the header with the title.
+- **Size**: Use one of the following values for the height of a left or right pane, and for the width of a top or bottom pane (on mobile, panes will always be full screen):
+	- `240px`
+	- `320px`
+	- `360px`
+	- `480px`
+	- `33%`
+	- `100%`
+- **Borderless**: Panes have borders to visually split content areas. Use this option to hide them if panes are placed in other layouts than just the main content where borders are disruptive.
+- **Position**: Panes can be positioned on the left, top, right or bottom.
+- **Layout**: Depending on which pane should have more focus the top/bottom or left/right panes can use more space.
+
+![Pane layouts](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=1681-28910&mode=design&t=iP7h44Wf17P209P7-4)
+
+1. Full height (left/right)
+2. Full width (top/bottom)
+
+## Behavior 
+
+- **Interaction**: The floating panes has a close button on the right side of the header; the inline pane has a collapse or expand button on the left side of the header.
```

Comment:
```suggestion
- **Interaction**: To expand floating panes, users typically click on a button or another interactive component within the main content. They can close floating panes by pressing on the button on the right side of the header which removed them from their view. Inline panes take a bar appearance when collapesd containing the title and the expand button. Users can expand inline panes by pressing on the expand button.
```
Another version: 
Floating panes have a close button on the right side of the header. Typically we place a button or another interactive element within the main content to trigger opening of floating panes. When expanded inline panes have a collapse button on the left side of the header. When collapsed, inline panes take a bar appearance containing the title and the expand button.